src/ui/database/common_event_widget.cpp

```diff
@@ -26,9 +26,14 @@ CommonEventWidget::CommonEventWidget(ProjectData& project, QWidget *parent) :
 {
 	ui->setupUi(this);
 
+	ui->comboTrigger->addItem("Disabled", 5);
+	ui->comboTrigger->addItem("Foreground", 3);
+	ui->comboTrigger->addItem("Background", 4);
```

Comment:
Use the constants from liblcf here and add a new Trigger lcf::rpg::EventPage::Trigger_disabled = 5

Reply:
So this requires a liblcf change, doesn't it? Adding the new trigger to the code makes the compiler complain about non-static data members.

Reply:
Found another enum in liblcf: `lcf::rpg::CommonEvent::Trigger_call` (5), `lcf::rpg::CommonEvent::Trigger_automatic` (3) and `lcf::rpg::CommonEvent::Trigger_call` (4).